include/dungeon_entity.h

```diff
@@ -2,6 +2,7 @@
 #define GUARD_DUNGEON_ENTITY_H
 
 #include "constants/move.h"
+#include "global.h"
```

Comment:
Why is `global.h` included here? I'd rather stick to including global in each `.c` file like `pokeemerald`

Reply:
Oh, this should be `constants/global.h`. `MAX_MON_MOVES` is defined there and used in the entity struct.

Reply:
Fixed now.